src/client.rs

```diff
@@ -105,6 +105,15 @@ pub trait ClientContext: Send + Sync {
         error!("librdkafka: {}: {}", error, reason);
     }
 
+    /// Refreshes the OAuth token.
+    ///
+    /// The default implementation just logs an error message at the `error`
+    /// log level. This function is called if/when the token refresh callback
+    /// is called.
+    fn refresh_oauth_token(&self, _client: *mut RDKafka, _oauthbearer_config: *const i8) {
```

Comment:
Maybe `outhbearer_config` should have a more friendly type, like a `&[u8]`?

Reply:
I'd be willing to make it `&str` or `String` since the oauthbearer configuration is supposed to be `key=value` pairs.

For example, this will convert into `&str`
```rust
let oauthbearer_config = CStr::from_ptr(oauthbearer_config).to_string_lossy();
context.refresh_oauth_token(client, oauthbearer_config.trim());
```

Reply:
`&str` sound good!